app/views/taxonomies/_loc_org_tabs.html.erb

```diff
@@ -1,10 +1,11 @@
 <% select_options ||= {} %>
 <% html_options ||= {} %>
+<% html_options[:tab_classes] ||= {} %>
 <% html_options[:location] ||= {} %>
 <% html_options[:organization] ||= {} %>
 
 <% if show_location_tab? %>
-  <div class="tab-pane" id="locations">
+  <div class="<%= "tab-pane #{html_options[:tab_classes][:location]}" %>" id="locations">
```

Comment:
This is equal and safes string interpolation, mainly it's bit more readable IMHO.
```suggestion
  <div class="tab-pane <%= html_options[:tab_classes][:location] %>" id="locations">
```